CONTRIBUTING.md

```diff
@@ -38,6 +38,7 @@ should not need deep knowledge of the system. [Have a look and see if anything s
 interesting](https://github.com/kubernetes-sigs/cluster-api/issues?q=is%3Aopen+is%3Aissue+label%3A%22good+first+issue%22).
 Alternatively, read some of the docs on other controllers and try to write your own, file and fix any/all issues that
 come up, including gaps in documentation!
+Also, before starting to work on the issue, make sure that it is not already taken up by someone else. The issues already taken up by someone have a [lifecycle/active](https://github.com/kubernetes-sigs/cluster-api/labels/lifecycle%2Factive) label. 
```

Comment:
What about moving this sentence before Alternatively? I think that it is more appropriate there...

Reply:
Also, what about adding the following sentence before line 47 so we are describing this practice also when someone starts to contribute to a patch
 
1. If working on an issue,  signal to the other contributes that you starting to work on it by applying the

Reply:
Sure.

Reply:
Can anyone apply this label by comenting `\lifecycle active` ?

Reply:
I made some changes :D